google-login-plugin/build.gradle

```diff
@@ -1,4 +1,5 @@
 // see https://github.com/JetBrains/gradle-intellij-plugin
+import org.apache.tools.ant.filters.ReplaceTokens
```

Comment:
No, I don't want to add this dependency at all. It feels heavyweight to pull in an entire extra third_party library for something we could probably do in a few lines of code ourselves.